website/content/en/docs/about/under-the-hood/guarantees.md

```diff
@@ -11,6 +11,28 @@ Here you can find an overview of delivery guarantee types and their meaning as w
 the stability of our components. Next, you can head over to the [components] page and use filters to
 see which components support specific guarantees.
 
+## Acknowledgement guarantees
```

Comment:
I think we should add a note to this section about how you can determine if the source and sink you are using support acknowledgements and what happens if the sink does not support them.

Reply:
Done in 3aa48fabb